pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentStickyKeyDispatcherMultipleConsumers.java

```diff
@@ -322,15 +322,9 @@ protected synchronized boolean trySendMessagesToConsumers(ReadType readType, Lis
             }
             // readMoreEntries should run regardless whether or not stuck is caused by
             // stuckConsumers for avoid stopping dispatch.
-            sendInProgress = false;
-            topic.getBrokerService().executor().execute(safeRun(this::readMoreEntries));
+            topic.getBrokerService().executor().schedule(safeRun(this::readMoreEntries), 100, TimeUnit.MILLISECONDS);
```

Comment:
Why the 100ms delay?

Reply:
because `sendingTaskCounter` will be decrement after the execution `trySendMessagesToConsumers`, I think it can replace `sendInProgress = false` in the line-325

Reply:
`readMoreEntries` is triggered in the method `sendMessagesToConsumers`， can we remove these two lines?

```java
topic.getBrokerService().executor().schedule(safeRun(this::readMoreEntries), 100, TimeUnit.MILLISECONDS);
```

Reply:
It seems that it doesn't need to be delayed execution, the `trySendMessagesToConsumers` is also within the synchronized block. I want to can be directly removed `sendInProgress = false`.